src/lib/blockchains/nxt.coffee

```diff
@@ -75,7 +75,7 @@ module.exports = (dnschain) ->
                     res.on 'end', () ->
                         try
                             response = JSON.parse data
-                            result.data = JSON.parse response.aliasURI
+                            result.data.value = JSON.parse response.aliasURI
```

Comment:
@toenu23 I don't think this will fix the issue...

It should be setting the `.data` property (we renamed it from `result.value`).